charts/govuk-apps-conf/templates/alert-rules/router.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: router-alert-rules
+  namespace: monitoring
+  labels:
+    release: prometheus
+    app: router
+spec:
+  {{- $.Files.Get "rules/router/alerts.yaml" | nindent 2}}
```

Comment:
What's the reason for not putting these inline? (similarly for the alert-rules one)

Reply:
what do you mean?

Reply:
Also, I deleted the other alert-rules.yaml, that was brought in when I cherry picked a commit, shouldn't have been there.

Reply:
Reason for not bringing them inline is to allow us to make changes in only 1 place to run the `promtool` unit tests and to deploy it using a helm chart.